### DISCUSSION TITLE
How to hide the popup after clicking it?

### DISCUSSION THREAD

**Discussion by younibugudano** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2677] [Q&A]
<materialDesign:PopupBox x:Name="BtnSpeed" Margin="0" DockPanel.Dock="Right" PlacementMode="TopAndAlignCentres"  StaysOpen="True" VerticalAlignment="Center" HorizontalAlignment="Center" Opacity="0.8" > 
                    <Grid Height="25" Width="220" Margin="0">
                        <Slider x:Name="SpeedSlider" Orientation="Horizontal" Width="180" Height="25" HorizontalAlignment="Stretch" VerticalAlignment="Stretch"  Margin="0,0,0,0" OverridesDefaultStyle="True" Style="{StaticResource Slider_CustomStyle}" ValueChanged="SpeedSlider_ValueChanged">
                        </Slider>
                    </Grid>
                </materialDesign:PopupBox>
![image](https://user-images.githubusercontent.com/34259213/48340253-191b1e00-e6a5-11e8-98dd-027ff333f242.png)

How to hide the popup after clicking it?
I tried trigger of MouseDown event and set ”StayOpen = true“.It didn't work.please help me!

**Comment by Keboo** [comment]
@younibugudano I am not sure I understand your question. I don't think you need to set the `StaysOpen` property. By default, the popup box will open, you will be able to adjust the slider, then simply clicking anywhere outside of the popup box will cause it to close. Is this what you are looking for?

**Comment by younibugudano** [comment]
Thanks so much for response. I understand your reply. Maybe my description is not very clear.I want to click the button to close the popup box when the popup box opens,but now when it was clicked,the popup box opened again.
The Button:
![image](https://user-images.githubusercontent.com/34259213/48384012-60e08a80-e723-11e8-90a9-0786a8515a4c.png)
